### DISCUSSION TITLE
Somtimes, old version installed from go-get

### DISCUSSION THREAD

**Discussion by hahwul** [hahwul/dalfox#178] [Q&A] [answered]
Hi, This is just my personal question. 
Sometimes the old version(1.2.0) is installed when installing with go-get, do you know why? 😭 
Version 1.2.x is appearing on these external sites, so I think there might be a problem with the repository or code. 

* https://libraries.io/go/github.com%2Fhahwul%2Fdalfox
* https://pkg.go.dev/github.com/hahwul/dalfox

If you know any issues, please share them!

cc) @bp0lr @rotemreiss @pdelteil @jsav0

**Comment by jsav0** [comment, marked as answer]
I think because the version bumped to v2+ , you'll have to make some adjustment to the repository structure maybe, see here <https://github.com/golang/go/wiki/Modules#semantic-import-versioning> or <https://blog.golang.org/v2-go-modules> maybe?

**Reply by hahwul** [reply]
Oh, thank you so much! I think what you said is the cause! I'll fix it!
